### PR TITLE
Add the nullified polling stations data [CSV]]

### PR DESCRIPTION
**Pull request by luigimorel**

### CODE REVIEW

File: results/2021/nullified_polling_stations/nullified_polling_stations.csv [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+79,45 SSEMBABULE,COUNTY,01 LUGUSULU,14 MBUYA,02 MBUYA MOSLEM,,478
+,,273 MAWOGOLA NORTH,,,,,
+80,45 SSEMBABULE,COUNTY,06 MITIMA,10 LWENTALE,02 KAGANGO N-Z,,513
+,,273 MAWOGOLA NORTH,,,,,
+81,45 SSEMBABULE,COUNTY,02 KAWANDA,08 LUTUNKU,01 KABULASOKE,,123
+,,,,,02 LUBANYI PRIMARY SCHOOL FRONT,,
+82,08 JINJA,042 KAGOMA COUNTY,06 NAMAGERA TOWN COUNCIL,25 LUBANYI WARD,YARD,,754
+83,08 JINJA,042 KAGOMA COUNTY,06 NAMAGERA TOWN COUNCIL,25 LUBANYI WARD,04 LUBANYI TECHNICAL,,359
+,,,,,,,"54,357"
+,,,Page 5 of 5,,,,
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations/nullified_polling_stations.csv]: @morelmiles Remove this page 5 of 5

File: results/2021/nullified_polling_stations/nullified_polling_stations.csv [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+65,45 SSEMBABULE,COUNTY,,02 KAWANDA,07 KYABI,06 NKOMA,436
+66,45 SSEMBABULE,119 LWEMIYAGA COUNTY,,03 KYEERA,10 LUBAALE,04 KAKINGA C.O.U,318
+,,273 MAWOGOLA NORTH,,,,,
+67,45 SSEMBABULE,COUNTY,,06 MITIMA,17 MITIMA,02 MITIMA,597
+68,51 SIRONKO,129 BUDADIRI COUNTY,WEST,10 MAFUDU,49 BUNGWANYI,01 BUNGWANYI C/U,337
+69,20 KOTIDO,094 JIE COUNTY,,03 KAMOR,37 NAADOI,04 KARUKADONG,207
+70,38 SOROTI,251 DAKABELA COUNTY,,05 KATINE,10 OIMAI,03 OLEL,588
+71,85 LAMWO,092 LAMWO COUNTY,,02 LOKUNG,57 PAWOR WEST,01 LATIDA CENTRE,573
+72,32 MUKONO,176 MUKONO COUNTY NORTH,,04 NAMA,07 KASENGE,08 MBALALA S.S.S COMPOUND [N-Z].,628
+,,,,Page 4 of 5,,,
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations/nullified_polling_stations.csv]: @morelmiles Remove this page 4 of 5

File: results/2021/nullified_polling_stations/nullified_polling_stations.csv [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+49,32 MUKONO,176 MUKONO COUNTY NORTH,02 KYAMPISI,14 KYABAKADDE,06 KYAMPISI SUBCOUNTY HQTRS,[NAK-Z],547
+,,062 KAWEMPE DIVISION,,,,,
+50,12 KAMPALA,NORTH,01 KAWEMPE DIVISION,08 KYEBANDO,02 SERINA PR. SCHOOL (A - K),,"1,177"
+51,32 MUKONO,176 MUKONO COUNTY NORTH,02 KYAMPISI,15 DDUNDU,01 KALAGALA,,967
+,,111 BUKOMANSIMBI NORTH,,,,,
+52,98 BUKOMANSIMBI,COUNTY,01 BIGASA,04 MBIRIZI,01 BIGASA SUBCOUNTY PLAY GROUND,,881
+53,32 MUKONO,176 MUKONO COUNTY NORTH,02 KYAMPISI,12 BULIJJO,04 NKONGE CHURCH,,537
+,,063 KAWEMPE DIVISION,,,,,
+54,12 KAMPALA,SOUTH,01 KAWEMPE DIVISION,02 BWAISE III,01 ST. JAMES CHURCH (A-M),,924
+,,,Page 3 of 5,,,,
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations/nullified_polling_stations.csv]: @morelmiles Remove this page 3 of 5

File: results/2021/nullified_polling_stations/nullified_polling_stations.csv [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+30,32 MUKONO,176 MUKONO COUNTY NORTH,04 NAMA,07 KASENGE,03 MBALALA S.S.S COMPOUND [A-M].,,499
+,,063 KAWEMPE DIVISION,,,,,
+31,12 KAMPALA,SOUTH,01 KAWEMPE DIVISION,02 BWAISE III,14 ST. FRANCIS NURSERY SCH. (O-Z),,474
+32,52 WAKISO,160 NANSANA MUNICIPALITY,02 BUSUKUMA DIVISION,14 KABUUMBA,02 KABUUMBA-KITAWULUZI,,771
+33,45 SSEMBABULE,119 LWEMIYAGA COUNTY,01 LWEMIYAGA,09 KAMPALA,01 BUGOROGORO,,732
+34,12 KAMPALA,067 RUBAGA DIVISION SOUTH,01 RUBAGA DIVISION,07 NDEEBA,13 B.M.K. (A-M),,"1,139"
+35,12 KAMPALA,067 RUBAGA DIVISION SOUTH,01 RUBAGA DIVISION,07 NDEEBA,21 B.M.K. (N-Z),,"1,087"
+,,062 KAWEMPE DIVISION,,,14 KISALOSALO (KIT-MUGE)-KYEBANDO,,
+36,12 KAMPALA,NORTH,01 KAWEMPE DIVISION,08 KYEBANDO,VOC. TRAINING CTRE,,"1,195"
+,,,Page 2 of 5,,,,
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations/nullified_polling_stations.csv]: @morelmiles Remove this page 2 of 5

File: results/2021/nullified_polling_stations/nullified_polling_stations.csv [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+12,119 KYOTERA,194 KYOTERA COUNTY,01 KABIRA,03 KYANIKA,04 BBANDA PRI. SCH,,752
+,,273 MAWOGOLA NORTH,,,,,
+13,45 SSEMBABULE,COUNTY,01 LUGUSULU,19 KAIRASYA,03 KIZAANO PENTECOSTAL CHURCH,,182
+,,273 MAWOGOLA NORTH,,,,,
+14,45 SSEMBABULE,COUNTY,01 LUGUSULU,22 MWITSI,04 NYAKATABO,,226
+15,36 RAKAI,249 BUYAMBA COUNTY,06 LWAMAGGWA,25 KIBUUKA,01 KIBUUKA P/SCHOOL,,469
+16,36 RAKAI,249 BUYAMBA COUNTY,06 LWAMAGGWA,25 KIBUUKA,02 KYANIKA CATHOLIC CHURCH,,564
+17,32 MUKONO,176 MUKONO COUNTY NORTH,04 NAMA,20 NAMAWOJJOLO,04 BWEFULUMYA EAST-AT FOREST HILL,,501
+18,12 KAMPALA,067 RUBAGA DIVISION SOUTH,01 RUBAGA DIVISION,07 NDEEBA,22 LATE,J.B. KASAJJA' S PLACE[N-Z],770
+,,,Page 1 of 5,,,,
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations/nullified_polling_stations.csv]: @morelmiles Remove this page 1 of 5

File: results/2021/nullified_polling_stations/nullified_polling_stations.csv [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+,,Presidential Election Nullified Polling Stations 2021,,,,,
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations/nullified_polling_stations.csv]: @morelmiles remove this duplicated header on each page

File: results/2021/nullified_polling_stations/nullified_polling_stations.csv [hunk 7]
```diff
@@ -0,0 +1,141 @@
+
+,District,Constituency,Scounty,Parish,,POLLING STATION,VOTERS
+No.,,,,,,,
+1,32 MUKONO,231 MUKONO MUNICIPALITY,01 GOMA DIVISION,05 SEETA WARD,31 GOSHEN LAND [NAK-Z],,823
+2,32 MUKONO,176 MUKONO COUNTY NORTH,02 KYAMPISI,14 KYABAKADDE,08 KASALA,,412
+3,32 MUKONO,176 MUKONO COUNTY NORTH,02 KYAMPISI,16 NTONTO,05 KASENENE,,419
+4,32 MUKONO,176 MUKONO COUNTY NORTH,04 NAMA,20 NAMAWOJJOLO,07 NAMAWOJJOLO ISLAMIC P/S [N-Z],,933
+5,32 MUKONO,176 MUKONO COUNTY NORTH,04 NAMA,20 NAMAWOJJOLO,08 NAMAWOJJOLO WEST [N-Z],,757
+,,062 KAWEMPE DIVISION,,,,,
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations/nullified_polling_stations.csv]: @morelmiles Remove these newlines which will cause issues when importing into other format

File: results/2021/nullified_polling_stations/nullified_polling_stations.csv [hunk 8]
```diff
@@ -0,0 +1,141 @@
+
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations/nullified_polling_stations.csv]: @morelmiles There is no need to add a folder for the nullified_polling_stations and rename the file to nullified_polling_stations_2020.csv

**Comment by ssmusoke** [comment]
@morelmiles apologies for the delay, I see duplicated records with blank details, Kawempe division etc, which need to be removed. 

I suggest that you open the file in excel and sort by the first column to see the details of the duplicates since the stations are not so many

File: results/2021/nullified_polling_stations.csv [hunk 9]
```diff
@@ -0,0 +1,124 @@
+
+,District,Constituency,Scounty,Parish,,POLLING STATION,VOTERS
+No.,,,,,,,
+1,32 MUKONO,231 MUKONO MUNICIPALITY,01 GOMA DIVISION,05 SEETA WARD,31 GOSHEN LAND [NAK-Z],,823
+2,32 MUKONO,176 MUKONO COUNTY NORTH,02 KYAMPISI,14 KYABAKADDE,08 KASALA,,412
+3,32 MUKONO,176 MUKONO COUNTY NORTH,02 KYAMPISI,16 NTONTO,05 KASENENE,,419
+4,32 MUKONO,176 MUKONO COUNTY NORTH,04 NAMA,20 NAMAWOJJOLO,07 NAMAWOJJOLO ISLAMIC P/S [N-Z],,933
+5,32 MUKONO,176 MUKONO COUNTY NORTH,04 NAMA,20 NAMAWOJJOLO,08 NAMAWOJJOLO WEST [N-Z],,757
+,,062 KAWEMPE DIVISION,,,,,
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations.csv]: @morelmiles This is an example of a duplicate record. This file should have 111 lines (including the header)

File: results/2021/nullified_polling_stations_2020.csv [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+75,119 KYOTERA,194 KYOTERA COUNTY,07 NABIGASA,35 KYASSIMBI,03 BULYANA MOSQUE (N-Z),,696
+76,45 SSEMBABULE,119 LWEMIYAGA COUNTY,02 BULONGO,05 BULONGO,08 KYATTUBA B,,318
+77,91 GOMBA,254 GOMBA WEST COUNTY,05 MADDU TOWN COUNCIL,26 MADDU WARD A,M),,632
+78,32 MUKONO,176 MUKONO COUNTY NORTH,04 NAMA,20 NAMAWOJJOLO,09 BWEFULUMYA WEST-AT FOREST HILL,,359
+79,45 SSEMBABULE,COUNTY,01 LUGUSULU,14 MBUYA,02 MBUYA MOSLEM,,478
+80,45 SSEMBABULE,COUNTY,06 MITIMA,10 LWENTALE,02 KAGANGO N-Z,,513
+81,45 SSEMBABULE,COUNTY,02 KAWANDA,08 LUTUNKU,01 KABULASOKE,,123
+82,08 JINJA,042 KAGOMA COUNTY,06 NAMAGERA TOWN COUNCIL,25 LUBANYI WARD,YARD,,754
+83,08 JINJA,042 KAGOMA COUNTY,06 NAMAGERA TOWN COUNCIL,25 LUBANYI WARD,04 LUBANYI TECHNICAL,,359
+,,,,,,,"54,357"
```
Comment (ssmusoke) [inline results/2021/nullified_polling_stations_2020.csv:86]: @morelmiles Remove this line

Comment (luigimorel) [reply results/2021/nullified_polling_stations_2020.csv:86]: @ssmusoke but the data is different.

**Pull request merged**